dev/fb/install-cli.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+vs_local_port=8081
+everest_svc_name="everest"
+everest_namespace="everest-system"
+everest_local_port=8080
+everest_remote_port=8080
+
+OS=$(uname -s | tr '[:upper:]' '[:lower:]')
+ARCH=$(uname -m | tr '[:upper:]' '[:lower:]')
+if [[ (${OS} == "linux" || ${OS} == "darwin") && ${ARCH} == "x86_64" ]]
+then
+	ARCH="amd64"
+elif [[ ${OS} == "linux" && ${ARCH} == "aarch64" ]]
+then
+	ARCH="arm64"
+fi
+
+# get the internal IP of the VS
+if nc -vz localhost ${vs_local_port} > /dev/null 2>&1 ; then
+  echo "Port forwarding for Percona Version Service is already set up."
+else
+  echo "Deploying Percona Version Service..."
+  # deploy VS and get its internal IP
+  . ./vs.sh
+  # sleep for 2 seconds to make sure the port-forward is ready
+  sleep 2
+fi
+
+# run everest installation with everest CLI
+echo "Installing Percona Everest version $(cat version.txt)..."
+./everestctl-${OS}-${ARCH} install \
+  --chart-dir "./helm-chart" \
+  --version "$(cat version.txt)" \
+  --version-metadata-url "http://localhost:${vs_local_port}"  \
+  --operator.mysql=true \
+  --operator.mongodb=true \
+  --operator.postgresql=true \
+  --skip-wizard \
+  --namespaces everest \
+  --helm.set "versionMetadataURL=http://percona-version-service.default.svc.cluster.local" \
+  "$@"
```

Comment:
Just a suggestion - same like our Tilt env, can we set the initial password to 'admin'? So that we don't have to do an extra step while testing?

Reply:
I thought about this but decided to keep as is because QA may need to use the functionality with default password.
Anyway you still can achieve the same result with 1 command only:
```
install-cli.sh --helm.set server.initialAdminPassword=admin
```

or 
```
install-helm.sh --set set server.initialAdminPassword=admin
```